tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/SetMembershipSimplifier.scala

```diff
@@ -12,7 +12,7 @@ import at.forsyte.apalache.tla.lir.values._
  * After Apalache's type-checking, we can rewrite some expressions to simpler forms. For example, the (after
  * type-checking) vacuously true `x \in BOOLEAN` is rewritten to `TRUE` (as `x` must be a `BoolT1`).
  *
- * We currently perform the following simplifications (for type-defining sets TDS, see [[isTypeDefining]]):
+ * We currently perform the following simplifications:
```

Comment:
We use the abbreviation TDS below, so we should keep this here:

```suggestion
 * We currently perform the following simplifications (for a definition of type-defining sets (TDS), see the private method `isTypeDefining`):
```